### PR TITLE
[API][Polyfill] support 'align_corners' in RESIZE_BILINEAR

### LINKED ISSUE #400: [API] Support 'align_corners' in RESIZE_BILINEAR op
By [definition](https://js.tensorflow.org/api/0.11.1/#image.resizeBilinear) of Tensorflow, `RESIZE_BILINEAR` has an extra parameter called `align_corners`. It's required by DeepLab models.

Without it, the output mask will have a slight bias towarding top-left corner.

__Without `align_corners`__
without_align

__With `align_corners`__
with_align

### PR DESCRIPTION
**Pull request by huningxin**
fix #400

### CODE REVIEW

File: src/nn/wasm/PreparedModel.js [hunk 1]
```diff
@@ -623,10 +623,23 @@ export default class PreparedModel {
...
         let input = operands[inputs[0]];
         let newHeight = operands[inputs[1]].value[0]; // Dont use newHeight and newWidth
         let newWidth = operands[inputs[2]].value[0];  // since outputShape has been set at first
+        // init resizeBilinearParams
+        // default set align_corners to false
+        let resizeBilinearParams = {
+          align_corners: false
+        }
+        if (inCount === 4) {
+          resizeBilinearParams.align_corners =
```
[the remark this answers is not in the document]
Comment (huningxin) [reply src/nn/wasm/PreparedModel.js]: Good suggestion. I will modify.

File: src/nn/webgl/WebGLModel.js [hunk 2]
```diff
@@ -365,7 +365,10 @@ export default class WebGLModel {
         const newHeight = operands[inputs[1]].value[0];
         const newWidth = operands[inputs[2]].value[0];
         const output = operands[outputs[0]];
-        const alignCorner = true;
+        let alignCorner = false;
+        if (inputs.length == 4) {
```
[the remark this answers is not in the document]
Comment (huningxin) [reply src/nn/webgl/WebGLModel.js]: Good suggestion. I will modify.

**Comment by huningxin** [comment]
@Wenzhao-Xiang , pushed a new version to address your comments. Please take another look. Thanks!

File: examples/util/tflite/TFliteModelImporter.js [hunk 3]
```diff
@@ -294,6 +294,7 @@ class TFliteModelImporter {
           inputs = [inputs[0]];
           inputs.push(this._addScalarInt32(newSize[0]));
           inputs.push(this._addScalarInt32(newSize[1]));
+          inputs.push(this._addScalarInt32(options.alignCorners() ? 1 : 0));
```
Comment (pinzhenx) [inline examples/util/tflite/TFliteModelImporter.js]: I don't know exactly if Android NNAPI would ignore this extra parameter. If not, NNAPI will fail on the non-atrous model.

Comment (huningxin) [reply examples/util/tflite/TFliteModelImporter.js]: Good point! In current implementation, NNAPI should report error. We can fix this by ignore the `align_corner` parameter in browser NNAPI backend. Filed https://github.com/intel/webml-polyfill/issues/468 for this.

**Comment by pinzhenx** [comment]
LGTM. Thank you for doing the polyfill part for us.

**Comment by Wenzhao-Xiang** [comment]
LGTM. Thanks!

**Comment by huningxin** [comment]
@Wenzhao-Xiang @pinzhenx , thanks for your review!

**Pull request merged**